index.js

```diff
@@ -5,20 +5,25 @@ const RUDY_DEFAULT = false
 const RATELIMITING_DEFAULT = false
 const LOGGING_DEFAULT = false
 const ERRORHANDLING_DEFAULT = false
+const USE_DYNAMIC_RATE_LIMITING_DEFAULT = false
+const USER_ACTIVE_TIMEOUT_DEFAULT = 30000
 
 const logSession = new Date().toISOString()
 const logStream = fs.createWriteStream('/tmp/express-requests-' + logSession + '.log', { flags: 'a' })
 
-const _interval = 3000 // milliseconds to check number of requests
-const _limit = 3 // limit for requests within interval
+const _interval = 10000 // milliseconds to check number of requests
+const _limit = 10 // limit for requests within interval
```

Comment:
10 requests/10 seconds is really slow and difficult to match when testing, should maybe changes to 3 req/seconds or something.

Reply:
When demonstrating, 10 seconds is a good time span considering that several users will have to connect to the server.